code/Master.py

```diff
@@ -1595,7 +1633,6 @@ def obj_expression(model):
 
         else:
             r = np.full((len(param["quantiles"]), len(settings)), 0)
-            no_sol_low_high = no_sol_low_high + reg + ', '
```

Comment:
why?

Reply:
I removed it cause it's not possible for a region to be both too high and too low